lms/templates/bookmarks/bookmarks_button.underscore

```diff
@@ -0,0 +1,5 @@
+<label for="course-bookmarks-button" class="sr"><%= gettext("Course") %></label>
```

Comment:
@muhammad-ammar a ```<label>``` element is not needed for ```<button>``` elements so you can remove this entire line.  The Accessible text for a ```<button>``` element is its text node.